Packs/CybleEvents/IncidentFields/incident_cybleeventstype.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
@cyble-dev Please use the common field "Event Type" instead.

Reply:
will go with the custom name as it would be easy to manage as one entity and tightly coupled

Reply:
@cyble-dev We are trying to minimize custom fields due to an overload we currently have.
The guidelines for incident fields is to use the common fields when it's possible and a lot of them are associated to different incident types (you can just associate them to your incident type and use it). Same for the other 2 fields I mentioned. If it's not possible for you to use the common fields for technical reasons, then you can create custom ones, but if the meaning and the usage of the custom field is the same as the common one, you should use the common field.

Reply:
yes the field type is different for one. Also had faced issue in fetching the assigned data from these fields into layout. With this technical hindrance faced while implementation, going for these custom fields.

Reply:
used common field for this